.github/release.yml

```diff
@@ -27,6 +24,9 @@ changelog:
     - title: ⬆️ Dependencies
       labels:
         - dependencies
+    - title: 📝 Documentation
```

Comment:
I think we probably want to have docs before dependencies. Usually the list of dependency updates is very long, making it hard to see the docs at the end of it.

Reply:
I was thinking that we should probably also re-order `change`, and `enhancement` in order of importance, something like:
```
    - title: 💣 Breaking Changes
      labels:
        - change
    - title: 🚀 Features
      labels:
        - enhancement
    - title: 🐛 Bug Fixes
      labels:
        - bug
    - title: 📦 Helm Chart
      labels:
        - helm_chart
    - title: 🧪 Tests
      labels:
        - tests
    - title: 🔨 Maintenance
      labels:
        - chore
    - title: 📝 Documentation
      labels:
        - documentation
    - title: ⬆️ Dependencies
      labels:
        - dependencies
    - title: Other Changes
      labels:
        - "*"
 ```
what do you think?

Reply:
I think this looks good. This puts 👀 directly on items that users should be aware of that might get looked over it is lower down the list.
brings the attention to any breaking changes and new features we implement.